xof/norx/sponge.go

```diff
@@ -0,0 +1,159 @@
+// Package norx implements a cryptographic sponge based on the experimental NORX cipher.
```

Comment:
Didn't we decide that norx will be left out of v1 (or moved to "experimental" status)?